variables.tf

```diff
@@ -365,8 +365,18 @@ variable "addons" {
   default     = {}
 
   validation {
```

Comment:
The current validations are fine for OCP-addon compatibility for versions `416`, `417` and `418`, but they do not restrict input to those versions. 

Please add an explicit validation to ensure that **only** versions `416`, `417`, or `418` are allowed.